Application/src/view/visuals/Draw.java

```diff
@@ -26,20 +27,29 @@
     private Canvas canvas_deployment; // Draws and handles class diagram graphical context.
 
     private ArrayList<Renderable> allClasses = new ArrayList<>(); // Stores the classes
+
+    private ArrayList<Renderable> allClassDiagramClasses = new ArrayList<>();
```

Comment:
name is self-explanatory but would be nice to add a comment for the sake of clarity